pkg/registry/servicecatalog/binding/storage.go

```diff
@@ -88,8 +88,17 @@ func Match(label labels.Selector, field fields.Selector) storage.SelectionPredic
 
 // toSelectableFields returns a field set that represents the object for matching purposes.
 func toSelectableFields(binding *servicecatalog.ServiceBinding) fields.Set {
+	// If you add a new selectable field, you also need to modify
+	// pkg/apis/servicecatalog/v1beta1/conversion[_test].go
 	objectMetaFieldsSet := generic.ObjectMetaFieldsSet(&binding.ObjectMeta, true)
-	return generic.MergeFieldsSets(objectMetaFieldsSet, nil)
+
+	specFieldSet := make(fields.Set, 1)
```

Comment:
I was about to comment on this being 3 instead of 1. Really, this should be `1 + len(objectMetaFieldSet`, and the function should be merging `objectMetaFieldSet` into `specFieldSet` instead of the other way around. However, that is beyond the scope of this PR.